test/integration/aaa_download_only_test.go

```diff
@@ -54,8 +51,7 @@ func TestDownloadOnly(t *testing.T) {
 	t.Run("group", func(t *testing.T) {
 		versions := []string{
 			constants.OldestKubernetesVersion,
-			constants.DefaultKubernetesVersion,
-			constants.NewestKubernetesVersion,
+			"v1.12.0",
```

Comment:
We need to download these versions to assert offline access in a later check.

How about just adding v1.12.0, and changing the test to check if there is a preload available, and altering it so that it checks for a preload file instead of stat'ing for docker image files?

Reply:
That makes sense, I ended up removing v1.12.0 because there is not preloaded tarball for oldest version right now.